lib/workers/repository/onboarding/branch/create.ts

```diff
@@ -12,18 +12,31 @@ export function createOnboardingBranch(
   logger.debug('createOnboardingBranch()');
   const contents = getOnboardingConfig(config);
   logger.debug('Creating onboarding branch');
-  let commitMessage;
-  // istanbul ignore if
-  if (config.semanticCommits) {
-    commitMessage = config.semanticCommitType;
+
+  let commitMessagePrefix = '';
+  if (config.commitMessagePrefix) {
+    commitMessagePrefix = config.commitMessagePrefix;
+  } else if (config.semanticCommits) {
+    commitMessagePrefix = config.semanticCommitType;
     if (config.semanticCommitScope) {
-      commitMessage += `(${config.semanticCommitScope})`;
+      commitMessagePrefix += `(${config.semanticCommitScope})`;
     }
-    commitMessage += ': ';
-    commitMessage += 'add ' + defaultConfigFile;
+  }
+  if (commitMessagePrefix) {
+    commitMessagePrefix += commitMessagePrefix.endsWith(':') ? ' ' : ':';
+  }
```

Comment:
Ideally the above logic would be put into a shared function if it is used identically in both onboarding and non-onboarding commit messages